src/charm_config.py

```diff
@@ -48,7 +48,15 @@ class Config:
     dnn: StrictStr = Field(default="internet", min_length=1)
     gnb_subnet: IPvAnyNetwork = Field(default="192.168.251.0/24")  # type: ignore
     access_interface_name: StrictStr = Field(default="eth0")
+    access_ip: str = Field(default="192.168.252.3/24")
```

Comment:
Shouldn't the default come from the charm config? Here we need to change the default in 2 places if we want to change it.

Reply:
This default (and also the others) are always overridden by the charm configuration with its default values: it will be sufficient to change them. I'll remove the defaults from here to avoid confusion.